### DISCUSSION TITLE
FLSUN SUPER RACER

### DISCUSSION THREAD

**Discussion by SkyGreeD1703** [supermerill/SuperSlicer#1951] [Q&A]
Hiiii 3d printer community 😁

first I wanted to tell you that I just started using your slicer and I love it ! 😍

currently i only own Ender 3's but on the 30th i will receive my new flsun super racer, i already looked in the profiles but it is not there yet :( 

do you think that the super racer will be able to be taken into account on your slicer?

**Comment by neophyl** [comment]
Profiles are added by users.  So unless a user that has one of the printers creates one and submits it then probably not.

**Reply by supermerill** [reply]
If it's your second printer, maybe you're savy enough to create and share one? ;-p
You can also search in the flsun community for a profile for SuSi, or for PrusSlicer (import it via the import-> from prusa entry. There may still be some things to adapt like the max g1 per second, but you should have the basic)

**Comment by coubi64** [comment]
Someone created one here : https://www.lesimprimantes3d.fr/forum/topic/44836-profils-superslicer-pour-flsun-super-racer-sr/

**Reply by dhstsw** [reply]
Hi,
I downloaded the .ini files.
If i try to import the pronter definition (SR-MG V2.ini) it adds it as a print profile, not as a printer.
What am i doing wrong?
thanks.
